src/lib/fragments/AgGrid.react.js

```diff
@@ -17,6 +17,7 @@ import 'ag-grid-community/styles/ag-grid.css';
 import 'ag-grid-community/styles/ag-theme-alpine.css';
 import 'ag-grid-community/styles/ag-theme-balham.css';
 import 'ag-grid-community/styles/ag-theme-material.css';
+import * as d3 from 'd3'
```

Comment:
The whole of D3 is pretty big (~300kb minified), and most of it is not useful for us. The pieces I can see wanting are `d3-format`, `d3-time-format`, and `d3-time`. `d3-array`? Do these functions have access to enough data to make `d3-array` useful?

Everything else seems too specialized or related to DOM manipulation so not useful to include here.

Reply:
I'll see if I can combine them, I think with the addition of the namespace functions for formatting, we should be fine to pair it down.